bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -293,7 +293,8 @@ static Set<LedgerFragment> splitIntoSubFragments(LedgerHandle lh,
             assert false;
         }
 
-        long numberOfEntriesToReplicate = (lastEntryId - firstEntryId) + 1;
+        long numberOfEntriesToReplicate = firstEntryId == INVALID_ENTRY_ID && lastEntryId == INVALID_ENTRY_ID ? 0 :
+                (lastEntryId - firstEntryId) + 1;
         long splitsWithFullEntries = numberOfEntriesToReplicate
```

Comment:
Can you share the configuration of `replicationEntryBatchSize` in the bookie server?
If I understand correctly, this problem will only occur when `replicationEntryBatchSize=1`

Reply:
Your understanding is correct. Our configuration is `rereplicationEntryBatchSize=1`